webserver/webserver/src/test/java/io/helidon/webserver/UnstableTempTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c)  2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.webserver;
+
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.ByteBuffer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.Flow;
+import java.util.function.Function;
+import java.util.jar.Attributes;
+import java.util.jar.JarEntry;
+import java.util.jar.JarOutputStream;
+import java.util.jar.Manifest;
+import java.util.logging.Logger;
+
+import io.helidon.common.http.DataChunk;
+import io.helidon.common.http.HashParameters;
+import io.helidon.common.http.Http;
+import io.helidon.common.reactive.Single;
+import io.helidon.media.common.MessageBodyWriterContext;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class UnstableTempTest {
+
+    private static final Logger LOGGER = Logger.getLogger(UnstableTempTest.class.getName());
+
+    private static final String JAR_NAME = "test.jar";
+    private static final String FILE_NAME = "test.js";
+    private static final String FILE_CONTENT = "alert(\"Hello, World!\");";
+    private static Path TEMP_DIR;
```

Comment:
Naming convention - this should be `tmpDir` or `tempDir` - all upper case is reserved for constants

Reply:
fixed